src/tool/subcommands/shed_cmd.rs

```diff
@@ -116,10 +116,15 @@ impl ShedCommands {
                     println!("{}", BASE64_STANDARD.encode(keypair_data));
                 }
             }
-            ShedCommands::Openrpc => {
+            ShedCommands::Openrpc { include } => {
```

Comment:
Could we have at least a simple component test that checks if the command succeeds (there are no panics) and that the output is a valid JSON? Along the lines of the ones here https://github.com/ChainSafe/forest/blob/main/tests/tool_tests.rs

Reply:
I think that would end up being just a more harnessed version of this ut, no? https://github.com/ChainSafe/forest/blob/main/src%2Frpc%2Fmod.rs#L517

Reply:
Theoretically, yes. In practice, this part of code may grow/change (for whatever reason) and it would be helpful to immediately catch obvious regressions here.

Reply:
I'll do it for you 😘, but since [this subcommand is semver-exempt anyway](https://github.com/ChainSafe/forest/blob/6433e238da4e97225c8ae8c36b6dad7402213c09/src/tool/subcommands/mod.rs#L74), I don't think it's necessary.

Reply:
Semver-exempt doesn't mean wild west - if a drunk developer puts less or more visible panic there, the CI should still fail.